src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5123,4 +5123,13 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_TypeReserved" xml:space="preserve">
     <value>The type name '{0}' is reserved to be used by the compiler.</value>
   </data>
+  <data name="ERR_FieldsInRoStruct" xml:space="preserve">
+    <value>Auto-implemented instance properties in readonly structs must be readonly.</value>
+  </data>
+  <data name="ERR_AutoPropsInRoStruct" xml:space="preserve">
+    <value>Field-like events are not allowed in readonly structs.</value>
```

Comment:
><value>Field-like events are not allowed in readonly structs.</value> [](start = 4, length = 69)

I do not understand what is going on here and why the tests are passing, but the messages between all three new errors still look mixed up.